From 476bbb6a4359e0c12e873dd3fa21c0d5ef8b1af4 Mon Sep 17 00:00:00 2001
From: WANG Xuerui <xen0n@gentoo.org>
Date: Wed, 19 Feb 2025 16:32:41 +0000
Subject: [PATCH 4/9] Bug 1948207 - Enable WebRTC for loongarch64 in
 toolkit/moz.configure. r=webrtc-reviewers,mjf

Differential Revision: https://phabricator.services.mozilla.com/D238171

(cherry picked from commit 4b88212bef8e20fb01c4d146a5a91b9ebf748ded)
---
 toolkit/moz.configure | 1 +
 1 file changed, 1 insertion(+)

diff --git a/toolkit/moz.configure b/toolkit/moz.configure
index a424446c7d446..b68315241f87a 100644
--- a/toolkit/moz.configure
+++ b/toolkit/moz.configure
@@ -1399,6 +1399,7 @@ def webrtc_default(target):
         "aarch64",
         "x86",
         "ia64",
+        "loongarch64",
         "mips32",
         "mips64",
         "ppc",
-- 
2.48.1

